packages/website/pages/docs/concepts/gateways.md

```diff
@@ -0,0 +1,96 @@
+# IPFS Gateways
+
+NFT.Storage uses the [InterPlanetary File System (IPFS)](https://ipfs.io) as a key part of its [storage and retrieval infrastructure][concepts-decentralized-storage].
+
+The IPFS network is a peer-to-peer network of computers that share resources to efficiently provide content to anyone that requests it. Computers that join the IPFS network use a protocol called Bitswap to request and supply blocks of data using a hash-based Content Identifier (CID) to uniquely identify each block.
```

Comment:
Should we add link to bitswap docs? https://docs.ipfs.io/concepts/bitswap/
This knowledge is not needed for using the gateway, but if we talk about it maybe we should point it for interested people to look into what happens behind the scenes